### ISSUE TITLE
Hard-coded deuterons

### ISSUE THREAD

**Issue by jhornung-gh** [denisbertini/mpi-sdf2opmd#3]
Hi Denis,

not sure if it is just there for testing but "opmd_reader_2d.cxx" contains hard-coded deuterons in line 146 and following.

Cheers,

Johannes

**Comment by denisbertini** [comment]
yes, the reader is for now just a "prototype"  to show how one could read output from openPMD converter. I used  `deuteron` species name in my example.
